src/app/components/views/homepage/homepage.component.html

```diff
@@ -35,8 +35,7 @@ <h4><b>Your Stuff (LMNO_USA_TECH)</b></h4>
   </div>
   <div class="row my-5">
     <div class="col-9 d-grid"> 
-      <!-- TODO: remove disabled flag from this once the job request system is implemented as a future feature. -->
-      <button type="button" class="btn btn-primary" (click)="onRequestJobButtonClick()" disabled>Request Job</button>
+      <button type="button" class="btn btn-primary" [disabled]="!jobsResponse" (click)="onRequestJobButtonClick()">Request Job</button>
```

Comment:
is `[disabled]="!jobsResponse"` a good way to disable this? Or would you rather there be logic (which would literally just be `this.disableRequestButton = !jobsResponse`)?

Reply:
I think I might argue that doing it the original way is a way to clearly show what's going on and why